client/me/profile/index.jsx

```diff
@@ -121,18 +113,26 @@ class Profile extends Component {
 							/>
 						</FormFieldset>
 
-						<FormFieldset
-							className={ clsx( {
-								'profile__gravatar-fieldset-is-loading': this.props.isFetchingUserSettings,
-							} ) }
-						>
-							<ToggleControl
-								disabled={ this.props.getDisabledState() || this.props.isFetchingUserSettings }
-								checked={ this.props.getSetting( 'gravatar_profile_hidden' ) }
-								onChange={ this.toggleGravatarHidden }
-								label={ <UpdatedGravatarString gravatarProfileLink={ gravatarProfileLink } /> }
-							/>
-						</FormFieldset>
+						<p className="profile__gravatar-profile-description">
+							<span>
+								{ this.props.translate(
+									'Your WordPress.com profile is powered by Gravatar. Your Gravatar is public by default and may appear on any site using Gravatar when you’re logged in with {{strong}}%(email)s{{/strong}}.' +
+										' To manage your profile and privacy settings, {{a}}visit your Gravatar profile here{{/a}}.',
```

Comment:
And other final tiny nit, I'd either remove the word "here" (ie. "visit your Gravatar profile") or I'd put the link on "here". It seems a bit unnecessary when the link is on the whole string. 

(You might also want to add a String Freeze label so that this can get some translations before merging)

Reply:
@Aurorum Thank you for the feedback. I have addressed it through [this commit](https://github.com/Automattic/wp-calypso/pull/93023/commits/a58353e765266b9a34099adc6fae155248f36899).

I will add the label, thanks :)